### DISCUSSION TITLE
Apple Music Official Lyrics Support

### DISCUSSION THREAD

**Discussion by Himalian** [ciderapp/Cider#1640] [Feature Request]
Lyrics from Apple are supported on Apple Music Web.So I hope cider can support lyrics from Apple in feature versions
![image](https://user-images.githubusercontent.com/97269989/213622117-8d27ad59-3a54-4e4c-af7a-62600fca7fa8.png)

**Comment by d3rpp** [comment]
This is already added in Cider 2 :smile:
